MVVM Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -57,4 +57,18 @@ extension FeedViewController {
 	private var feedImagesSection: Int {
 		return 0
 	}
+	
+	var errorMessage: String? {
+		(tableView.tableHeaderView as? ErrorView)?.message
```

Comment:
Use `errorView?.message` to simplify the code.

```suggestion
		errorView?.message
```